<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSettings</class>
 <widget class="QWidget" name="VideoSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>116</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QComboBox" name="m_pRate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Rate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="m_pChannel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="m_pResolution">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_pDevice"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pDevice</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>VideoSettings</receiver>
   <slot>slotDeviceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pChannel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>VideoSettings</receiver>
   <slot>slotChannelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pResolution</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>VideoSettings</receiver>
   <slot>slotResolutionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRate</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>VideoSettings</receiver>
   <slot>slotRateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotReloadDevices()</slot>
  <slot>slotDeviceChanged(int)</slot>
  <slot>slotChannelChanged(int)</slot>
  <slot>slotResolutionChanged(int)</slot>
  <slot>slotRateChanged(int)</slot>
 </slots>
</ui>
